### ISSUE TITLE
X11 font configuration

### ISSUE THREAD

**Issue by verdurin** [hpcugent/easybuild-easyconfigs#3070]
In testing an update of `RELION`, I found that the font configuration provided by the X11-related easyconfigs was missing, with the result that it wouldn't start.

The ugly fix for this was to do:

`yum groupinstall x11 fonts`

There is a scriptlet in the `fontconfig` RPM which creates the font cache:

```
postinstall scriptlet (using /bin/sh):
/sbin/ldconfig

umask 0022

mkdir -p /var/cache/fontconfig

# Force regeneration of all fontconfig cache files
# The check for existance is needed on dual-arch installs (the second
#  copy of fontconfig might install the binary instead of the first)
# The HOME setting is to avoid problems if HOME hasn't been reset
if [ -x /usr/bin/fc-cache ] && /usr/bin/fc-cache --version 2>&1 | grep -q 2.10.95 ; then
  HOME=/root /usr/bin/fc-cache -f
fi
```

Should we add this to the `fontconfig` easyconfig?

**Comment by boegel** [comment]
Well, can we? Which path should we use?

@hajgato Thoughts?

**Comment by hajgato** [comment]
@boegel @verdurin We have a a font cache for `fontconfig` at `$EBROOTFONTCONFIG/var/cache/fontconfig`. So that could be used. The bigger problem is that we use the system provided fonts, and we do not install them, so they are used from the system. At my system, only the fonts in the `/usr/share/fonts`  directory are used, and the `/usr/share/X11/fonts` directory not. 
I think we have to do the following:
1) install fonts. We have some free fonts, like gnu-free and the DejaVu family. We also have to install X11 fonts. (like some fixed width fonts, helvetica, and some chinese fonts)
2)Make sure the `fontconfig` uses all fonts we have installed. (scans all the directories during the install)

I hope I did not misunderstood the question.

**Comment by verdurin** [comment]
No, that's a great summary, @hajgato, and I agree with your suggestions.

**Comment by hajgato** [comment]
3) We have to make sure that when `fontconfig` is loaded, the right `fonts.conf` is used (and not the system one from `/etc/fonts/fonts.conf` although I do not know how we can do that.
